gcc/rust/lex/rust-codepoint.h

```diff
@@ -22,6 +22,8 @@
 #include "rust-system.h"
 
 namespace Rust {
+
+// FIXME: move this to rust-unicode.h?
 struct Codepoint
```

Comment:
I think this class should be moved to rust-unicode.h. 
At least it should not be put in under gcc/rust/lex/.